arches/app/views/api/controlled_lists.py

```diff
@@ -0,0 +1,424 @@
+            )
+
+        return JSONResponse(status=HTTPStatus.NO_CONTENT)
+
+    def delete(self, request, **kwargs):
+        item_id: UUID = kwargs.get("id")
+        objs_deleted, unused = ControlledListItem.objects.filter(pk=item_id).delete()
+        if not objs_deleted:
+            return JSONErrorResponse(status=HTTPStatus.NOT_FOUND)
+        return JSONResponse(status=HTTPStatus.NO_CONTENT)
+
+
+@method_decorator(
+    group_required("RDM Administrator", raise_exception=True), name="dispatch"
+)
+class ControlledListItemValueView(View):
+    def add_new_value(self, request):
+        data = JSONDeserializer().deserialize(request.body)
+
+        value = ControlledListItemValue(
+            controlled_list_item_id=UUID(data["item_id"]),
+            valuetype_id=data["valuetype_id"],
+            language_id=data["language_id"],
+            value=data["value"],
+        )
+        try:
+            value.full_clean()
+        except ValidationError as ve:
+            return JSONErrorResponse(
+                message="\n".join(ve.messages), status=HTTPStatus.BAD_REQUEST
+            )
+        value.save()
+
+        return JSONResponse(value.serialize(), status=HTTPStatus.CREATED)
+
+    def post(self, request, **kwargs):
+        if not (value_id := kwargs.get("id", None)):
+            return self.add_new_value(request)
+
+        data = JSONDeserializer().deserialize(request.body)
+
+        try:
+            value = ControlledListItemValue.values_without_images.get(pk=value_id)
+        except ControlledListItemValue.DoesNotExist:
+            return JSONErrorResponse(status=HTTPStatus.NOT_FOUND)
+
+        value.value = data["value"]
+        value.valuetype_id = data["valuetype_id"]
+        try:
+            value.language = Language.objects.get(code=data["language_id"])
+        except Language.DoesNotExist:
+            return JSONErrorResponse(status=HTTPStatus.NOT_FOUND)
+
+        try:
+            value.full_clean()
+        except ValidationError as ve:
+            return JSONErrorResponse(
+                message="\n".join(ve.messages), status=HTTPStatus.BAD_REQUEST
+            )
+        value.save()
+
+        return JSONResponse(value.serialize())
+
+    def delete(self, request, **kwargs):
+        value_id = kwargs.get("id")
+        try:
+            value = ControlledListItemValue.values_without_images.get(pk=value_id)
+        except:
+            return JSONErrorResponse(status=HTTPStatus.NOT_FOUND)
+
+        try:
+            value.delete()
+        except ValidationError as ve:
+            return JSONErrorResponse(
+                message="\n".join(ve.messages), status=HTTPStatus.BAD_REQUEST
+            )
+        return JSONResponse(status=HTTPStatus.NO_CONTENT)
+
+
+@method_decorator(
+    group_required("RDM Administrator", raise_exception=True), name="dispatch"
+)
+class ControlledListItemImageView(View):
+    def add_new_image(self, request):
```

Comment:
Could this go anywhere else? Maybe folded into `post`?

Reply:
I agree that these methods might be better served on the model itself EG: 
ControlledListItemImage.add_new(id, type, value)
which wouldn't try to catch any errors but instead allow the POST method to catch them.